ansible_collections/arista/avd/plugins/action/validate_and_template.py

```diff
@@ -10,7 +13,7 @@
 from ansible_collections.arista.avd.plugins.plugin_utils.schema.avdschema import AristaAvdError, AvdSchema
 from ansible_collections.arista.avd.plugins.plugin_utils.utils import compile_searchpath, template
 
-VALID_CONVERSION_MODES = ["disabled", "warning", "info", "debug"]
+VALID_CONVERSION_MODES = ["disabled", "warning", "info", "debug", "quiet"]
```

Comment:
should it be documented: https://avd.sh/en/devel/docs/input-variable-validation-BETA.html?h=disabled+warnin#validation-options ?

Reply:
ah yes I forgot that doc. Updated two others.

Reply:
fixed